electron/src/util/electronUtils.ts

```diff
@@ -0,0 +1,191 @@
+"use strict";
```

Comment:
Some utils that I copied (and in some cases modified) from our PXT.Util module. I can't use PXT.Util directly because I don't want to have to depend on files outside the electron repo, which would make it rather annoying when building and packaging the app.